specs/data_structures.md

```diff
@@ -540,27 +558,23 @@ Validator objects represent all the information needed to be keep track of a val
 1. `Unbonded`: This validator has completed its unbonding and has withdrawn its stake. The validator object will remain in this status until `delegatedCount` reaches zero, at which point it is destroyed.
 
-## Delegation
+In the validators subtree, validators are keyed by the [hash](#hashdigest) of their [address](#address). The first byte is then replaced with `VALIDATORS_SUBTREE_ID`. By construction, the validators subtree will be a subset of a mirror of the [accounts subtree](#account).
 
-```C++
-enum DelegationStatus : uint8_t {
-    Bonded = 1,
-    Unbonding = 2,
-};
-```
+## ActiveValidatorCount
 
-| name              | type                        | description                                         |
-| ----------------- | --------------------------- | --------------------------------------------------- |
-| `status`          | `DelegationStatus`          | Status of this delegation.                          |
-| `validator`       | [Address](#address)         | The validator being delegating to.                  |
-| `stakedBalance`   | `uint64`                    | Delegated stake, in `4u`.                           |
-| `beginEntry`      | [PeriodEntry](#periodentry) | Entry when delegation began.                        |
-| `endEntry`        | [PeriodEntry](#periodentry) | Entry when delegation ended (i.e. began unbonding). |
-| `unbondingHeight` | `uint64`                    | Block height delegation began unbonding.            |
+| name            | type     | description                  |
+| --------------- | -------- | ---------------------------- |
+| `numValidators` | `uint32` | Number of active validators. |
 
-Delegation objects represent a delegation. They have two statuses:
-1. `Bonded`: This delegation is enabled for a `Queued` _or_ `Bonded` validator. Delegations to a `Queued` validator can be withdrawn immediately, while delegations for a `Bonded` validator must be unbonded first.
-1. `Unbonding`: This delegation is unbonding. It will remain in this status for at least `UNBONDING_DURATION` blocks, and while unbonding may still be slashed. Once the unbonding duration has expired, the delegation can be withdrawn.
+Since the [validator set](#validator) is stored in a Sparse Merkle Tree, there is no compact way of proving that the number of active validators exceeds `MAX_VALIDATORS` without keeping track of the number of active validators. There is only a single leaf in the active validator count subtree, which is keyed with zero (i.e. `0x0000000000000000000000000000000000000000000000000000000000000000`), and the first byte replaced with `VALIDATOR_COUNT_SUBTREE_ID`.
```

Comment:
imo this should be part of validator subtree: 
https://github.com/lazyledger/lazyledger-specs/pull/32/commits/f1cba758138d074b7a18b59d62b3236587f920d0#r439773622

Reply:
[Unlike moving the inactive validator set into the accounts subtree](#discussion_r439783834), I might be okay with moving the validator count into the active validators subtree because it's guaranteed to be in a fixed location.

Reply:
I think it semantically belongs to the validator subtree and also a single value tree seems weird.

Reply:
Alright, will migrate the count into the active validators subtree.

Reply:
Fixed in 4d87996.